### PR TITLE
Add flags.org

### PR DESCRIPTION
**Pull request by RasPhilCo**

### CODE REVIEW

**Comment by RasPhilCo** [comment]
keep'n it 💯

**Comment by RasPhilCo** [comment]
@dickeyxxx I updated this after you approved to handle the 'flag or org required' piece

File: src/flags/team.js [hunk 1]
```diff
@@ -8,9 +8,11 @@ export default function TeamFlag (options: Options = {}, env: typeof process.env
   const defaultOptions: Options = {
     char: 't',
     description: 'team to use',
-    parse: (input) => {
+    parse: (input, cmd) => {
       if (input) return input
       if (envTeam) return envTeam
+      let org = cmd ? cmd.flags.org : undefined
+      if (org) return org
```
Comment (dickeyxxx) [inline src/flags/team.js]: I would write this:

```js
if (cmd && cmd.flags.org) return cmd.flags.org
```

File: src/flags/org.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import {type Flag} from 'cli-engine-command/lib/flags'
+
+type Options = $Shape<Flag<string>>
+export default function OrgFlag (options: Options = {}, env: typeof process.env = process.env): Flag<string> {
+  const envTeam = env.HEROKU_ORGANIZATION
+  const defaultOptions: Options = {
+    char: 'o',
+    description: 'org to use',
```
Comment (dickeyxxx) [inline src/flags/org.js]: no description needed, make this hidden by default

**Pull request merged**